R/texreg.R

```diff
@@ -1925,6 +1925,467 @@ print.texregTable <- function(x, ...) {
+    } 
+    
+    # aggregate confidence intervals or SEs
+    if (isTRUE(use.se[i]) && length(models[[i]]@se) > 0) {
+        lower.inner <- dataframe$co - dataframe$se
+        upper.inner <- dataframe$co + dataframe$se
+        lower.outer <- dataframe$co - 2 * dataframe$se
+        upper.outer <- dataframe$co + 2 * dataframe$se
+    } else if (length(models[[i]]@ci.low) == 0 && length(models[[i]]@se) > 0) {
+        z.inner <- qnorm(1 - ((1 - 0.5) / 2))
+        lower.inner <- dataframe$co - (z.inner * dataframe$se)
+        upper.inner <- dataframe$co + (z.inner * dataframe$se)
+        z.outer <- qnorm(1 - ((1 - ci.level) / 2))
+        lower.outer <- dataframe$co - (z.outer * dataframe$se)
+        upper.outer <- dataframe$co + (z.outer * dataframe$se)
+    } else if (length(models[[i]]@se) == 0 && length(models[[i]]@pvalues) > 0) {
+        stop("Model has p-values but no SEs. SEs or CIs are required for plotting.")
+    } else {
+        lower.outer <- dataframe$ci.low
+        upper.outer <- dataframe$ci.up
+        lower.inner <- rep(0, length(dataframe$co.names))
+        upper.inner <- rep(0, length(dataframe$co.names))
+    }
+    
+    if (length(lower.outer) > 0) {
+        dataframe <- cbind(dataframe, lower.outer)     
+    }
+    if (length(lower.inner) > 0) {
+        dataframe <- cbind(dataframe, lower.inner)     
+    }
+    if (length(upper.outer) > 0) {
+        dataframe <- cbind(dataframe, upper.outer)     
+    }    
+    if (length(upper.inner) > 0) {
+        dataframe <- cbind(dataframe, upper.inner)     
+    }
+    
+    # which terms are significant?
+    if (length(dataframe$pv) == 0 & length(dataframe$ci.low) == 0) {
+        stop("Impossible to estimate significance since no CIs or SEs are provided.")
+    }
+    
+    if (length(dataframe$pv) > 0) {
+        signif.outer <- dataframe$pv < (1 - ci.level)
+    } else if (length(dataframe$ci.low) > 0) {
+        signif.outer <- ((dataframe$ci.low > 0 & dataframe$ci.up > 0) | (dataframe$ci.low < 0 & dataframe$ci.up < 0))
+    }
+    
+    if (is.logical(signif.outer) && length(signif.outer) == length(dataframe$co)) {
+        signif <- sapply(signif.outer, isTRUE)
+    } else if (sapply(signif.outer, isTRUE)) {
+        signif <- apply(cbind(lower.outer, upper.outer), 1,
+                        function(x) x[1] <= 0 && x[2] >= 0)
+    } else if (sapply(signif.outer, isFALSE)) {
+        signif <- apply(cbind(lower.inner, upper.inner), 1,
+                        function(x) x[1] <= 0 && x[2] >= 0)
+    } else {
+        stop("'signif.outer' does not correspond to the intervals provided.")
+    }
+    
+    
+    dataframe <- cbind(dataframe, signif)
+    
+    if (length(co) == 0) {
+        stop(paste("No coefficients available. Was the 'omit.coef' argument", 
+                   "misspecified? If coefficients were renamed using the",
+                   "'custom.coef.names' argument, 'omit.coef' refers to the renamed",
+                   "coefficients."))
```

Comment:
Can you double-check if this order is true?

Reply:
yes it works!